MetaMask/metamask-extension
# test:e2e :Add Blockaid e2e test - ERC721 "trade_order_farming" UI warning

## **Description**
Adds e2e tests for PPOM Blockaid Security Alert for trade order farming warning. 
In this test, we mock the `eth_call` infura calls. I also noted that `eth_debugTraceCall` was never sent when calling this using Blockaid's example.

## **Related issues**

Fixes: [#1455](https://github.com/MetaMask/MetaMask-planning/issues/1455)

## **Manual testing steps**

To see the calls that were mocked
1. Launch MetaMask
2. Open the background.html console
3. Open Network tab and start recording network calls
4. Open test-dapp and click on trade order farming button
5. Goto Network tab and stop recording.
6. You should see a couple of calls to infura.


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
